api/src/services/collections.ts

```diff
@@ -16,7 +16,7 @@ import { Accountability, FieldMeta, RawField } from '@directus/shared/types';
 export type RawCollection = {
 	collection: string;
 	fields?: RawField[];
-	meta?: Partial<CollectionMeta>;
+	meta?: Partial<CollectionMeta> | null;
```

Comment:
`meta` is already optional/undefinable. Would it be possible to stick to making either `null` or `undefined` the "empty" value?

Reply:
The "Raw" collection is what the user submits on creation of the collection, in which case "meta" could either be null _or_ undefined. We'll have to expect both coming in from the user. In the regular "Collection" type, "meta" can't be undefined 👍🏻